examples/charts/area.py

```diff
@@ -1,6 +1,7 @@
 from collections import OrderedDict
-from bokeh.charts import Area
-from bokeh.plotting import show, output_file
+
+from bokeh.charts import Area, show, output_file
```

Comment:
I'll confess to being torn on this one. It's really, really nice from  user's perspective to just have all the needed tools in one place. I am not really sure there is any negative consequence for us on the dev side. I would say that maybe we do move the functions to their own "io" api module, and make a point to always import the entire "io" api module into the other interfaces.